packages/hl7v2-server/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "hl7v2-server",
```

Comment:
ditto: if we're updating it to MLLP, update here and the package's name

Reply:
addressed